docs/source/index.rst

```diff
@@ -6,36 +6,58 @@
 Substra documentation
 =====================
 
-Substra is a framework offering distributed orchestration of machine learning tasks among partners while guaranteeing secure and trustless traceability of all operations.
+Substra is an open source federated learning framework. It provides a flexible python interface to run federated learning training at scale.
 
-- :doc:`Getting started <get_started/overview>`
+Substra main usage is a production usage. It has already been deployed and used by hospitals and biotech companies. Yet Substra can also be used on a single machine on a virtually splitted dataset for two use cases:
 
-  An overview and guide to install Connnect python library.
+* to debug code before launching experiments on a real network
+* to perform FL simulations
 
-- :doc:`Documentation <documentation/concepts>`
 
-  In depth documentation of Substra concepts and API reference.
+Substra was created by Owkin and is now hosted by the Linux Foundation for AI and Data.
 
-- :doc:`Examples <auto_examples/index>`
+Interfaces
+^^^^^^^^^^
 
-  Examples of usage of Substra.
 
-- :doc:`Releases notes <additional/release>`
+Substra has four user interfaces:
+    * **Substra**: a low-level Python library (also called SDK). Substra is used to create datasets, algorithms and machine learning tasks on the platform.
+    * **Substrafl**: a high-level federated learning Python library. Substrafl is used to run complex federated learning experiments at scale.
```

Comment:
Maybe add something like "Substrafl is based on Substra and abstract complexity for the user" (to rephrase), to make it more explicit than low-level and high-level. Otherwise I don't really see the link between the 2 libraries and "Substrafl is used to run complex federated learning experiments at scale" + the fact that it comes second makes me think it is to go further, for expert users.

Reply:
agree, done!